pymysqlreplication/tests/base.py

```diff
@@ -84,6 +84,13 @@ def isMySQL8014AndMore(self):
             return True
         return version == 8.0 and version_detail >= 14
 
+    def isMySQL8016AndMore(self):
+        version = float(self.getMySQLVersion().rsplit(".", 1)[0])
```

Comment:
Any reasons for opting to cast to float instead of int?

Reply:
Because Other code converts version into float I really don't know
Can I investigate this problem?

Reply:
Yes. I believe casting to `int` is enough.
We should apply Boy Scout Rule.
Please investigate.

Reply:
My bad. I mistook the number of split tokens.